docs/content/docs/start/examples.md

```diff
@@ -17,3 +17,45 @@ The sticky notes [brainstorming app](https://github.com/microsoft/FluidExamples/
 shows how simple experiences become compelling when you make them collaborative. This example uses distributed
 maps (SharedMaps) to update the state of sticky notes as well as keep track of who added which ideas and who
 voted for those ideas.
+
+## Collaborative text area
+
+The [collaborative text area app](https://github.com/microsoft/FluidExamples/tree/main/angular-demo) shows
+how to create a text area that can be collaboratively edited by multiple clients. It uses React to create
+the view. See also [Building a collaborative TextArea]({{< relref "docs/recipes/collaborative-text-area.md" >}}).
+
+## Separating the view from the Fluid business logic
+
+The [multiframework dice roller app](https://github.com/microsoft/FluidExamples/tree/main/multi-framework-diceroller)
+shows you how to keep your view layer separate from your Fluid layer. By changing a single line
+of code, you can switch between views based on React, Vue, Web Components, and simple, no framework
+JavaScript. See also [Using Fluid with React]({{< relref "docs/recipes/react.md" >}}),
+[Using Fluid with Vue]({{< relref "docs/recipes/vue.md" >}}),
+and [Using Fluid with Web Components]({{< relref "docs/recipes/web-components.md" >}})
+
+## Angular and Fluid
+
+The [Angular timestamp app](https://github.com/microsoft/FluidExamples/tree/main/angular-demo) shows how clients
+can share a timestamp. Use it as a starter template to build your own Fluid Framework application on the
+Angular framework. See also [Using Fluid with Angular]({{< relref "docs/recipes/angular.md" >}}).
+
+## React and Fluid
+
+The [React dice app](https://github.com/microsoft/FluidExamples/tree/main/react-starter-template) shows how
+to make incorporate Fluid state into a React-based app. Use it as a starter template to implement a 
+React-based view for your own Fluid Framework application.
+
+The [React timestamp app](https://github.com/microsoft/FluidExamples/tree/main/react-demo) shows how clients
+can share a timestamp. See also [Using Fluid with React]({{< relref "docs/recipes/react.md" >}}).
```

Comment:
Sharing timestamp may not be a primary goal here: https://github.com/microsoft/FluidExamples/tree/main/react-demo#fluid-examplereact-demo

Reply:
fixed